core/templates/dev/head/components/exploration_welcome_modal_directive.html

```diff
@@ -0,0 +1,61 @@
+<script type="text/ng-template" id="modals/welcomeExploration">
+  <div class="modal-body oppia-long-text">
+    <h1>Welcome!</h1>
+    <p>
+      Looks like you're new to creating explorations.
+    </p>
+    <p>
+      Would you like an introduction to creating in Oppia?
+    </p>
+    <a class="oppia-welcome-dismiss" ng-click="cancel()">No, take me to the editor.</a>
```

Comment:
I suggest adding a blank href attribute (`<a href class="..." ...>`), otherwise you can't reach this link by pressing the Tab key.